NitroxClient/Helpers/NitroxProtobufSerializer.cs

```diff
@@ -13,13 +14,15 @@ public class NitroxProtobufSerializer
         public readonly Dictionary<Type, int> NitroxTypes = new Dictionary<Type, int>();
 
         private readonly Dictionary<Type, int> knownTypes;
+        private readonly HashSet<string> componentWhitelist;
```

Comment:
What is this component whitelist used for, it doesn't seem to be used?

Reply:
Component whitelist is used in UWE ProtobufSerializer, this just simply adds Nitrox components to that serializer to stop some unnecessary logging